#ATCP Added list call (#18906)
microsoftgraph/microsoft-graph-docs: concepts/custom-security-attributes-examples.md

HTTP/1.1 204 No Content
```

## List custom security attribute assignments

> ### Example 1: List a single-valued custom security attribute assigned to a user

Reviewer: "Get" over "List" for showing /users/{id}. List is more suitable for /users collection.
```suggestion
### Example 1: Get the custom security attributes assigned to a user
```

> The following example shows how to list a custom security attribute assignment assigned to the user.

Reviewer: ```suggestion
The following example shows how to list a custom security attribute assignment assigned to the user. The custom security attribute is a multi-string value with the following settings:
```

- Attribute set: `Engineering`
- Attribute: `datacenter`

> - Attribute value: `Redmond`

Reviewer: Let's show that this is a multi-string value
```suggestion
- Attribute data type: Collection of Strings
- Attribute value: ["Redmond"]
```

#### Request

<!-- {
  "blockType": "request",

>   "name": "customsecurityattribute_remove_from_user_singlevalue"

Reviewer: ```suggestion
  "name": "customsecurityattribute_get_for_user"
```

<!-- {
  "blockType": "request",

>   "name": "customsecurityattribute__from_user_singlevalue"

Reviewer: Maintain the example name for traceability
```suggestion
  "name": "customsecurityattribute_remove_from_user_singlevalue"
```
